### DISCUSSION TITLE
How to Make Money on GitHub

### DISCUSSION THREAD

**Discussion by xuuhu** [community/community#65255] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want to write projects on GitHub to make money, how do I do it?

**Comment by leikang123** [comment]
First of all, you need to activate the tipping function on the project, then bind the bank card, open it and view it in the project settings, github sponrnrs This setting can be turned on.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
